server/db/seeds.js

```diff
@@ -50,8 +50,11 @@ function generateOodlets(quantity, oodlers, quantifiedThingies) {
   console.log(`Generating ${quantity} Oodlets...`);
 
   for (let i = 0; i < quantity; i++) {
+    let randCreateDate = faker.date.past();
```

Comment:
How about this:
`let notSoRandCreateDate = moment().subtract(2*quantity, 'weeks').toDate()`

That way we'll get a nice simulation of a real Oodlet backlog